cmd/minikube/cmd/start.go

```diff
@@ -106,11 +125,16 @@ func runStart(cmd *cobra.Command, args []string) {
 		NetworkPlugin:     viper.GetString(networkPlugin),
 		ExtraOptions:      extraOptions,
 	}
+	fmt.Println("Using Kubernetes Version:", kubernetesConfig.KubernetesVersion)
 	if err := cluster.UpdateCluster(host, host.Driver, kubernetesConfig); err != nil {
 		glog.Errorln("Error updating cluster: ", err)
 		cmdUtil.MaybeReportErrorAndExit(err)
 	}
 
+	config.Set(kubernetesVersion, kubernetesConfig.KubernetesVersion)
```

Comment:
Is this better to change to `config.SetKubernetesConfig(kubernetesConfig)` & handle the saved mappings in there rather than here?